scripts/Dockerfile.devel

```diff
@@ -14,8 +14,10 @@ COPY build.env /
 RUN source /build.env \
     && \
     ( test -n "${GOARCH}" && exit 0; echo -e "\n\nMissing GOARCH argument for building image, install Golang or run: make containerized-build GOARCH=amd64\n\n"; exit 1 ) \
-    && mkdir -p /usr/local/go \
-    && curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-${GOARCH}.tar.gz | tar xzf - -C ${GOROOT} --strip-components=1
+    && mkdir -p ${GOROOT} \
+    && curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-${GOARCH}.tar.gz | tar xzf - -C ${GOROOT} --strip-components=1 \
+    && go version \
+    && true
```

Comment:
Out of curiosity, what’s the benefit of `true` here?

Reply:
The `true` on the 1st and last line reduces the number of changed lines (adding `\` on the previous like) and helps with improving the output of `git blame`. There is no need to change a previous or next line, adding new lines do not touch others. It is a convention recommended in some guidelines for Dockerfiles/Containerfiles.

Unfortunately I can't find the source for this recommendation anymore...

Reply:
Wow. That is something that I did not know. Thanks!!